### DISCUSSION TITLE
chrome_crashpad_handler and stacking

### DISCUSSION THREAD

**Discussion by curiosityseeker** [roddhjav/apparmor.d#347] [Ideas]
Similarly as with Firefox, 

`@{lib_dirs}/chrome_crashpad_handler  rpx,`

in `abstractions/app/chromium` is causing an error. And as in `abstractions/app/firefox`  stacking solves the problem:

`@{lib_dirs}/chrome_crashpad_handler  rpx -> brave//&brave-crashpad-handler,`

However, that abstractions should also work for other Chromium-based browsers, so this rule should better be generalized. I'm not sure how to do this. What do you think?

**Comment by roddhjav** [comment]
This can be done with: `@{profile_name}//&brave-crashpad-handler,`

**Comment by curiosityseeker** [comment]
Yes, but other Chromium-based browsers won't use **brave**-crashpad-handler.

**Comment by curiosityseeker** [comment]
Ah, possibly with

`@{profile_name}//&@{profile_name}-crashpad-handler,`

EDIT: Just tried it with Brave but didn't work.

**Comment by curiosityseeker** [comment]
So, neither 

`@{profile_name}//&@{profile_name}-crashpad-handler,`

nor 

`@{profile_name}//&brave-crashpad-handler,`

worked, only the syntax in the first post.

**Comment by roddhjav** [comment]
I moved the discussion as an issue as it is a bug: #348
